java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/window/WindowComputer.scala

```diff
@@ -226,6 +238,11 @@ class WindowComputer(config: WindowAggConfig, jit: HybridSeJitWrapper, keepIndex
     SparkRowUtil.getLongFromIndex(config.orderIdx, orderField.dataType, curRow)
   }
 
+  def extractUnsafeKey(curRow: UnsafeRow): Long = {
```

Comment:
returns scala Long here, so `null` from `unsafeGetLongFromIndex ` will return 0. Is it ok?

Reply:
It is the same logic of old code. We may re-consider the logic of handling null data later.